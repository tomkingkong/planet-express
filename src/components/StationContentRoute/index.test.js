import React from 'react';
import { shallow } from 'enzyme';

import { StationContentRoute } from '.';

describe('StationContentRoute component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<StationContentRoute />);
    expect(wrapper).toMatchSnapshot();
  });
});