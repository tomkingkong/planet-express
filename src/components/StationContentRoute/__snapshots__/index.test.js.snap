// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StationContentRoute component should match snapshot 1`] = `
<Switch>
  <Route
    component={[Function]}
    path="/ISS/destinations"
  />
  <Route
    component={[Function]}
    path="/ISS/store"
  />
</Switch>
`;
